test-d/jsonify.ts

```diff
@@ -328,5 +331,5 @@ declare const objectWithAnyProperties: Jsonify<Record<string, any>>;
 expectType<Record<string, any>>(objectWithAnyProperties);
 
 /// #629
```

Comment:
To keep this in line with other comments in the file

```suggestion
// Regression test for https://github.com/sindresorhus/type-fest/issues/629
```